CHANGELOG.md

```diff
@@ -1,5 +1,105 @@
+- Remove invariant on nested sibling **VirtualizedLists** without unique listKey props; they now trigger a **RedBox** ([af5633b](https://github.com/facebook/react-native/commit/af5633b))
+- **FlatList** and **VirtualizedList**'s default `keyExtractor` now checks `item.id` and `item.key` ([de0d7cf](https://github.com/facebook/react-native/commit/de0d7cf) by [@sahrens](https://github.com/sahrens))
+- Slightly speedup core initialization by moving native version check to DEV only ([5bb2277](https://github.com/facebook/react-native/commit/5bb2277) by [@mmmulani](https://github.com/mmmulani))
+- `react-native-community/eslint-config` is now at v0.0.5 ([2ef56c0](https://github.com/facebook/react-native/commit/2ef56c0) by [@matt-oakes](https://github.com/matt-oakes))
+- `react` is now at v16.8.1 ([ee681b7](https://github.com/facebook/react-native/commit/ee681b7) and [6001acb](https://github.com/facebook/react-native/commit/6001acb) by [@mdvacca](https://github.com/mdvacca) and [@gaearon](https://github.com/gaearon))
+- `eslint-plugin-react-native` is now at v3.7.0 ([2becfab](https://github.com/facebook/react-native/commit/2becfab) by [@monotv](https://github.com/monotv))
+- `react-native-community/cli` is now at v2.0.0-alpha.20 ([0815f81](https://github.com/facebook/react-native/commit/0815f81) by [@thymikee](https://github.com/thymikee))
+- `flow` is now at v0.98 ([0e1dfd4](https://github.com/facebook/react-native/commit/0e1dfd4) by [@nmote](https://github.com/nmote))
+- `prettier` is now at v1.17.0 ([ff9f8f3](https://github.com/facebook/react-native/commit/ff9f8f3))
+- `metro` packages are now at v0.54.1 ([7ff3874](https://github.com/facebook/react-native/commit/7ff3874), [343f0a1](https://github.com/facebook/react-native/commit/343f0a1) by [@motiz88](https://github.com/motiz88))
+- TODO add mention of AndroidX *BREAKING*
+
+#### Android specific
+
+- Move message queue initialization in ReactContext into its own method ([db40c28](https://github.com/facebook/react-native/commit/db40c28) by [@ejanzer](https://github.com/ejanzer))
+- Use class canonical name for `PARTIAL_WAKE_LOCK` tag ([88dbb45](https://github.com/facebook/react-native/commit/88dbb45) by [@timwangdev](https://github.com/timwangdev))
+
+#### iOS specific
+
+- *BREAKING*: Split React.podspec into separate podspecs for each Xcode project; your libraries will need to update for this change as well to avoid CocoaPods build errors ([2321b3f](https://github.com/facebook/react-native/commit/2321b3f) by [@fson](https://github.com/fson))
+- **SectionList**'s `scrollToLocation` on iOS now counts `itemIndex` like Android; both platforms are now consistent ([248a108](https://github.com/facebook/react-native/commit/248a108) by [@vonovak](https://github.com/vonovak))
```

Comment:
I think this actually isn't just ios-specific but affects both platforms, and is possible breaking for some people (if they care about index being x or x + 1). I need to add more context here, will try to do this soon. Ping me if I don't please.

Reply:
hey @vonovak sorry to ping you already on this, but we are planning on releasing this week so I'd love to know the more context :D

Reply:
Hey @vonovak from my read it still seems to be platform specific; if you have any specific guidance please let me know.

Reply:
@turnrye @kelset sorry for silence; you're right that this is ios-specific in the sense it patches https://github.com/facebook/react-native/commit/f66e8ebf51a04f47eaa2406bce461ef8f8f34d0a which made it into 0.59 (while I thought it didn't).

Let me write it in a way that hopefully allows you to copy-paste it into the changelog: 

`scrollToLocation` on iOS now counts `itemIndex` like Android; both platforms are now consistent AND the behavior in 0.60 on both platforms is different from how it was before 0.59.

let us consider this snippet:

```js
sectionListRef.scrollToLocation({
  animated: false,
  itemIndex: 0,
  sectionIndex: 0,
  viewPosition: 0,
  viewOffset: 0,
});
```

pre-0.59, this would result into the section list scrolling to the first item and the first section heading would NOT be visible.

in 0.60, this results into scrolling to the first section heading, with the first item visible below it. 

[optional]
Therefore, if you want to keep the old behavior of scrolling to the item, you will need to conditionally add 1 to `itemIndex` so that  your code scrolls to the item (and not the heading) while making sure you prevent https://github.com/facebook/react-native/issues/18098 from happening.

Reply:
Thank you for the detailed explanation!